privacy-config/privacy-config-api/src/main/java/com/duckduckgo/privacy/config/api/DrmBlock.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2021 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.privacy.config.api
+
+/** Public interface for the DRM blocking feature */
+interface DrmBlock {
```

Comment:
I'd move all the new drm stuff and put in the site-permissions module. Privacy config is only for parsing so we are moving all the features away from it.